lib/tpm2.c

```diff
@@ -1,13 +1,10 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
 
-#include <tss2/tss2_esys.h>
 #include <tss2/tss2_mu.h>
 
 #include "log.h"
 #include "object.h"
-#include "tpm2.h"
```

Comment:
keeping the <thisfile>.h on includes is a good idea ensure alignment of function declarations in header and implementations.

Reply:
I am not sure why deheader remove this header.
I will add them back and commit again, Thanks.